docs/modules/ROOT/pages/how-to-guides/testing_applications/proc_creating_custom_test.adoc

```diff
@@ -91,6 +91,28 @@ spec:
 . Add your new custom test as an integration test in {ProductName}.
 .. For additional instructions on adding an integration test, see this document: xref:how-to-guides/testing_applications/proc_adding_an_integration_test.adoc[Adding an integration test].
 
+.Data injected into integration PipelineRun
+
+In the integration test created above, integration-service adds the following Parameters/Labels automatically to the integration PipelineRun.
+
+Parameters:
+
+ . `SNAPSHOT` containing the https://github.com/redhat-appstudio/application-api/blob/main/api/v1alpha1/snapshot_types.go[Snapshot] as a json string
```

Comment:
Just an idea: maybe we can have the example contents of the Snapshot stored in our [integration-examples](https://github.com/redhat-appstudio/integration-examples) repository and link to it here to provide the users with a better idea of how it will look.

Reply:
Do you mean we store the example content of the Snapshot [here](https://github.com/redhat-appstudio/integration-examples/blob/main/pipelines/integration_test_app.yaml#L13C3-L13C3)? I just wonder if we need to change the pipeline/task in the page also?

Reply:
We could have a new directory called `examples` with a more well defined example of the Snapshot like the one you mentioned earlier:
```
{"application":"application-sample","components":[{"name":"component-sample","containerImage":"quay.io/redhat-appstudio/sample-image@sha256:34fb1c72bc976937d0a59849160746cec12a8c925f31e74dc9f3b922f74da0e9","source":{"git":{"url":"[git@github.com](mailto:git@github.com):redhat-appstudio/integration-service.git\n","revision":"sha256:3703843e80f39188c060a6ed0f524519a682be1e7141fee29b878b1973e9783c"}}
```

Reply:
@dirgim, so, I just need to create a new file contains the example content of Snapshot , right?

Reply:
Yes, I think that would do the job.